recipes/xmsinterp/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "3.0.5" %}
+
+package:
+  name: xmsinterp
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aquaveo/xmsinterp/archive/{{ version }}.tar.gz
+  sha256: f97efecccf2f708e55a960c19c368326ef72747b1d9f79acee3845801d1d4ce4
+
+build:
+  number: 0
+  skip: True  # [not py36]
+  merge_build_host: true
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    # just to avoid build/host merge issues
+    # but these should be in host.
+    - zlib
+    - xmscore
+    - xmsgrid
+    - python
+    - pybind11
+    - boost-cpp
+    - pip
+  run:
+    - xmscore
+    - xmsgrid
+    - python
+    - numpy
```

Comment:
This should be built as listed here: https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy

Reply:
I believe that we need that only if this against `numpy`, right?

Reply:
Ahh yeah, sorry.  I thought it was!